plugins/build.gradle.kts

```diff
@@ -17,7 +20,11 @@ subprojects {
     tasks.test {
         useJUnitPlatform()
         testLogging {
-            events("passed", "skipped", "failed")
+            exceptionFormat = TestExceptionFormat.FULL
+            events(TestLogEvent.SKIPPED, TestLogEvent.FAILED)
+            showExceptions = true
+            showCauses = true
+            showStackTraces = true
```

Comment:
Please note: this change hides `PASSED` tests, which should be more clean than now